Gamecube/menu/FileBrowserFrame.cpp

```diff
@@ -472,6 +473,17 @@ void fileBrowserFrame_LoadFile(int i)
 		int ret = loadISO( &dir_entries[i] );
 		
 		if(!ret){	// If the read succeeded.
+			if(Autoboot){
+				// Wiimpathy:
+				// FIXME: The MessageBox is a hacky way to fix input not responding.
+				// No time to improve this...
+				menu::MessageBox::getInstance().setMessage("Autobooting game...");
```

Comment:
This is extremely hacky and I don't like it. Just add a new method in the MessageBox class, why do you need to do it in setMessage() at all?

Reply:
https://github.com/emukidid/pcsxgc/pull/32/commits/f7ae144eb86236038af18a10f0c5d40113623457 removes this hacky way and replaces it with a sleep command instead.